src/main/java/io/appium/java_client/android/AndroidMobileOptions.java

```diff
@@ -0,0 +1,1716 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.android;
+
+import io.appium.java_client.remote.AndroidMobileCapabilityType;
+import io.appium.java_client.remote.MobileOptions;
+import io.appium.java_client.remote.MobilePlatform;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.time.Duration;
+import java.util.List;
+
+public class AndroidMobileOptions extends MobileOptions<AndroidMobileOptions> {
+
+    /**
+     * Creates new instance with platformName specified for Android.
+     */
+    public AndroidMobileOptions() {
+        setPlatformName(MobilePlatform.ANDROID);
+    }
+
+    /**
+     * Creates new instance with provided capabilities capabilities.
+     *
+     * @param source is Capabilities instance to merge into new instance
+     */
+    public AndroidMobileOptions(Capabilities source) {
+        this();
+        merge(source);
+    }
+
+    /**
+     * Set the Timeout in milliseconds used to wait for adb command execution.
+     *
+     * @param ms is the number of milliseconds to wait for adb command execution.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#ADB_EXEC_TIMEOUT
+     */
+    public AndroidMobileOptions setAdbExecTimeout(Duration ms) {
```

Comment:
Please change the arg name to `duration` and mention in docs that the minimum time resolution unit is one millisecond. Same for other caps, which use duration - they need to be time-unit agnostic, since the Duration entity already implements when we need.

Reply:
Ah, but capabilities doesn't, I need to convert it. 
664223287e7973d5c78c320bc331d084d4c91239